### DISCUSSION TITLE
无法从个人主页打开系列视频

### DISCUSSION THREAD

**Discussion by wwhdgithub** [ywmoyue/biliuwp-lite#471] [bug]
### Bug 描述

在哔哩哔哩晚会官方账号主页点击视频“第一篇章：日落 全程精彩【2023 B站跨年晚会精彩全程】”等系列视频无法打开

### 操作步骤

```plain text
1.搜索哔哩哔哩晚会
2.找到默认第一个视频的“第一篇章：日落 全程精彩【2023 B站跨年晚会精彩全程】”，作者为哔哩哔哩晚会，点击进入
3.进入视频页面后点击作者区域进入主页
4.点击任意系列类的视频，均弹出未知错误
```

### 期望结果

可正常打开视频并播放

### 截图

![截图](https://github.com/ywmoyue/biliuwp-lite/assets/86828383/9fff87d4-fdf7-413b-ba8c-936a2fe74d67)

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
产生此bug的原因可能与 #452 一致。
